darshan-util/pydarshan/darshan/cli/summary.py

```diff
@@ -15,7 +15,7 @@
 import darshan
 import darshan.cli
 from darshan.backend.cffi_backend import accumulate_records
-from darshan.lib.accum import log_get_bytes_bandwidth, log_file_count_summary_table
+from darshan.lib.accum import log_file_count_summary_table, module_overview_table
```

Comment:
It seems to me like you should delete `log_get_bytes_bandwidth()` if you're not using it anymore, along with the testing for it.